docgen/generate-docs.js

```diff
@@ -72,59 +72,54 @@ function runTypedoc() {
  * Moves files from subdir to root.
  * @param {string} subdir Subdir to move files out of.
  */
-function moveFilesToRoot(subdir) {
-  return exec(`mv ${docPath}/${subdir}/* ${docPath}`)
-    .then(() => {
-      exec(`rmdir ${docPath}/${subdir}`);
-    })
-    .catch(e => console.error(e));
+async function moveFilesToRoot(subdir) {
+  await exec(`mv ${docPath}/${subdir}/* ${docPath}`)
+  await exec(`rmdir ${docPath}/${subdir}`);
 }
 
 /**
  * Renames files to remove the leading underscores.
  * We need to do this because devsite hides these files.
- * Example: 
+ * Example:
  * _cloud_functions_.resource.html => cloud_functions_.resource.html
  */
-function renameFiles() {
-  return fs.readdir(docPath).then(files => {
-    console.log(files);
-    files.forEach(file => {
-      let newFileName = file;
-      if (_.startsWith(file, "_") && _.endsWith(file, "html")) {
-        newFileName = _.trimStart(file, "_");
-        fs.rename(`${docPath}/${file}`, `${docPath}/${newFileName}`, (err) => {
-          if (err) console.log(err)
-        });
-      }
-    })
-  })
+async function renameFiles() {
+  const files = await fs.readdir(docPath);
+  console.log(files);
+  const renames = [];
+  files.forEach(file => {
```

Comment:
Nit since it was already like this: https://google.github.io/styleguide/tsguide.html#iterating-containers suggests that we should avoid .foreach and use for loops instead.